src/main/java/com/github/kaktushose/jda/commands/entities/JDACommandsBuilder.java

```diff
@@ -244,7 +285,7 @@ public JDACommands build() {
         try {
             settings = YamlLoader.load(JDACommands.class.getClassLoader().getResource("settings.yaml"));
             log.debug("Found a settings file. Maybe overriding given runtime values");
-        } catch (IOException | NullPointerException ignore) {
+        } catch (IOException | NullPointerException | ClassNotFoundException ignore) {
```

Comment:
`ClassNotFoundException` has nothing to do with a missing settings file. No reason to ignore it